shardingsphere-db-protocol/shardingsphere-db-protocol-mysql/src/test/java/org/apache/shardingsphere/db/protocol/mysql/constant/MySQLServerInfoTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.shardingsphere.db.protocol.mysql.constant;
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class MySQLServerInfoTest {
+
+    private final String SPECIFIC_VERSION = "5.1.47";
+
+    private final String EXPECTED_VERSION = "5.1.47-ShardingSphere-Proxy 5.0.0-RC1";
+
+    private final String EXPECTED_NULL_VERSION = "null-ShardingSphere-Proxy 5.0.0-RC1";
```

Comment:
Please do not use static fields in it can be inline in the test cases

Reply:
I don't quite understand `static field` . does ss recommand not to use class level variable in test case?
I push the code again , is that what ss expected ? please check , thx